pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/ui/player/settings/PlayerSettingsViewModel.kt

```diff
@@ -130,6 +134,8 @@ class PlayerSettingsViewModel(
         }
     }.stateIn(viewModelScope, SharingStarted.WhileSubscribed(), emptyList())
 
+    private val currentPlaybackMetrics = if (player is PillarboxExoPlayer) player.currentMetricsAsFlow() else emptyFlow()
```

Comment:
If `player` is not a `PillarboxExoPlayer`, we use an `emptyFlow()`. Which means that the `combine` for `settings` below will never execute, since it's missing a value.
```suggestion
    private val currentPlaybackMetrics = if (player is PillarboxExoPlayer) player.currentMetricsAsFlow() else flowOf(null)
```

Reply:
Yes and we don't care of this value if it is not a `PillarboxExoPlayer` in that case by default it is null.